components/AuthorBox.tsx

```diff
@@ -28,9 +47,16 @@ const AuthorBox: React.ElementType<AuthorBoxProps> = ({
   axios
     .get("https://wp.stanforddaily.com/wp-json/tsd/v1/authors/" + id + "/")
     .then(response => {
+      console.log(response);
```

Comment:
Let's remove this before merging

Reply:
Just the console.log right? Just took that out.

Reply:
Yup